src/equations/laplace_diffusion_2d.jl

```diff
@@ -28,8 +28,8 @@ end
 # TODO: parabolic; should this remain in the equations file, be moved to solvers, or live in the elixir?
 # The penalization depends on the solver, but also depends explicitly on physical parameters,
 # and would probably need to be specialized for every different equation.
-function penalty(u_outer, u_inner, equations_parabolic::LaplaceDiffusion2D, dg::ViscousFormulationLocalDG)
-  return dg.penalty_parameter * (u_outer - u_inner) * equations_parabolic.diffusivity
+function penalty(u_outer, u_inner, inv_h, equations_parabolic::LaplaceDiffusion2D, dg::ViscousFormulationLocalDG)
+  return dg.penalty_parameter * (u_outer - u_inner) * equations_parabolic.diffusivity * inv_h
 end
```

Comment:
Didn't you mention that multiplication by `inv_h` is a bug?

Reply:
Yes - good point. It's not tested anywhere yet, but it is technically incorrect. Fixed in https://github.com/trixi-framework/Trixi.jl/pull/1400/commits/a8c0933bd0077c45372c3937c40fce772e348886